logo.js

```diff
@@ -174,9 +174,25 @@ function LogoInterpreter(turtle, stream, savehook)
 
   // Returns a Promise that will resolve after yielding control to the
   // event loop.
+  var lastTimeYeilded = Date.now();
```

Comment:
Spelling: should be `lastTimeYielded`.

Reply:
(and update the references to the variable)

Reply:
Ouch, I keep messing this word up... 😃